packages/core-data/src/entity-provider.js

```diff
@@ -275,7 +275,7 @@ export function useEntityBlockEditor( kind, name, { id: _id } = {} ) {
 			const edits = { blocks: newBlocks, selection };
 			registry.batch( () => {
 				updateFootnotes( edits.blocks );
-				editEntityRecord( kind, name, id, edits );
+				editEntityRecord( kind, name, id, edits, { isCached: false } );
```

Comment:
@youknowriad I rebase this PR to try it with footnotes, but undo still doesn't seem batched. Do we need to update the `editEntityRecord` call in `useEntityProp` as well?

Reply:
That change is not correct. For this PR to only create one undo level for footnotes, you need one of the `editEntityRecord` calls (either footnotes one or actual change) to be `isChached: true` (isCached: false is the default)

Reply:
Oh, sorry. I just rebased with the previous change.